scripts/controlnet.py

```diff
@@ -62,9 +62,11 @@
 switch_values_symbol = '\U000021C5' # ⇅
 camera_symbol = '\U0001F4F7'        # 📷
 reverse_symbol = '\U000021C4'       # ⇄
+scissors_symbol = '\U00002702'      # ✂
 
 webcam_enabled = False
 webcam_mirrored = False
+crop_enabled = False
```

Comment:
Having this global means all units of both txt2img and img2img will have the same state. Consider instantiating an `gr.State` to pass around state to gradio component updates.

Reply:
I didn't know about that, thanks.